nodejs/eks/examples/nodegroup/iam.ts

```diff
@@ -0,0 +1,27 @@
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+
+const managedPolicyArns: string[] = [
+    "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy",
+    "arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy",
+    "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly",
+];
```

Comment:
Possibly for a follow-up PR - but I wonder if it would be helpful to export this array from the package so this doesn't have to be hard-coded into the user-code?   

We could even add a helper that does the same as the `createRole` below as a convenience thing - like `eks.createDefaultInstanceRole`?